src/Controls/samples/Controls.Sample.UITests/Controls.Sample.UITests.csproj

```diff
@@ -18,7 +18,11 @@
     <_FastDeploymentDiagnosticLogging>True</_FastDeploymentDiagnosticLogging>
     <WindowsPackageType>None</WindowsPackageType>
   </PropertyGroup>
-
+  
+  <ItemGroup>
+    <PackageReference Include="Mopups" Version="1.3.1" />
```

Comment:
Do we really need add this? can this functionality be mocked in a different way ?

Reply:
Need to verify. Probably can also reproduce it with a modal page.

Reply:
Cannot reproduce the issue with a modal ContentPage. We can keep the sample using Popup or remove it and leave just the fix without the sample.

Feedback?

Reply:
For now, because we are not morroring the Mopups package, removed the sample.